guidelines/component-usage.md

```diff
@@ -1 +1,29 @@
+## Table of content
 
+1. [Syntax](#syntax)
+2. [Component Structure](#component-structure)
+
+## Syntax
+
+1. #### Naming Style **[⬆](#table-of-content)**
+
+   * Always use the `.js` extension.
+   * Use UpperCamelCase as a components name.
+   * Use lowerCamelCase for prop names.
+   * Component name should be the same as the file name. This means that the use of `index` files is forbidden for components. It's easier to find components by filename rather than by folder name.
+   * Component filenames should be in UpperCamelCase. It's allowed to use dots (`.`) to explicitly mention type of component: `Dashboard.component.js`.
```

Comment:
What's the point of the dot notation? Also, you already say "Use UpperCamelCase as a components name" and "[The] Component name should be the same as the file name", so isn't this redundant?

Reply:
Makes sense. Removed